tgui/packages/tgui/styles/interfaces/CloningConsole.scss

```diff
@@ -0,0 +1,55 @@
+@use '../colors.scss';
+@use '../components/Button.scss';
+
+.Cloning-Console_FlexTable {
```

Comment:
BEM convention is two delimiters between block and element, so would be `...__FlexTable` rather than `..._FlexTable`. Not requiring for this PR, would appreciate if you could clean up.